src/main/java/appeng/helpers/DualityInterface.java

```diff
@@ -480,6 +490,7 @@ protected void addToCraftingList(final ItemStack is) {
                     this.craftingList = new LinkedList<>();
                 }
 
+                details.setPriority(36 * this.getPriority() - slot);
```

Comment:
I think the priority is calculated way too eager causing the priority to sky-rocket to 36 * the interfaces priority.
i suggest the following
```suggestion
                details.setPriority(this.getPriority() + 35 - slot);
```

Reply:
`this.getPriority` refers to the priority of the interface, not the pattern itself. Since there are up to 36 patterns in an interface it should have a weight of 36.